config/opal_check_pmi.m4

```diff
@@ -358,9 +363,9 @@ AC_DEFUN([OPAL_CHECK_PMIX],[
            LDFLAGS=$opal_external_pmix_save_LDFLAGS
            LIBS=$opal_external_pmix_save_LIBS
 
+           opal_external_pmix_CPPFLAGS="-I$pmix_ext_install_include_dir"
```

Comment:
we do __not__ want `-I/usr/include` in `CPPFLAGS`

Reply:
@ggouaillardet I can filter that case, sure.  When the prefix is `/usr`, that var might be `/usr/include` or `/usr/include/pmix`.